src/Microsoft.ML.Transforms/Text/LdaStaticExtensions.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.TextAnalytics;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.Transforms.Text
```

Comment:
>Microsoft.ML.Transforms.Text [](start = 10, length = 28)

Pete will ask you to put static extensions into Microsoft.ML.StaticPipe;

Reply:
I sure will

---
In reply to: [234771139](https://github.com/dotnet/machinelearning/pull/1410#discussion_r234771139) [](ancestors = 234771139)